Makefile

```diff
@@ -46,3 +46,8 @@ installdeps-test:
 installdeps-dev:
 	pip install -e . -q
 	pip install -r dev-requirements.txt -q
+
+.PHONY: installdeps-docs
+installdeps-docs:
+	pip install -e . -q
+	pip install -r docs-requirements.txt -q
```

Comment:
This is used by the circleci `build_docs` code [at the top](https://github.com/alteryx/evalml/pull/1561/files#r543735032) and isn't part of the bugfix itself.